config/bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require dirname(__DIR__) . '/vendor/autoload.php';
+
+// Load cached env vars if the .env.local.php file exists
+// Run "composer dump-env prod" to create it (requires symfony/flex >=1.2)
```

Comment:
If this is needed it should live in the Dockerfile. But... I tried localy and it dumps the env vars into an array file to improve the performance but there're some problems here:
- This cached config file MUST be generated before app start
- Containers are immutable and is the orchestrator who binds the config before run it via volume mounts from secrets or config maps, or via env vars... Following this this command must be called at preStart but, why the need of composer in production? So I spent some time playing with it and I don't like workflow. At all, it's so hacky, and indicates that there's a problem in the workflow. This file can be generated as part of the cache, generated at the first requests and that's it and outside composer. 
I ended opening an issue https://github.com/symfony/flex/issues/512
At the moment, red light.

Reply:
Bugazo https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/bootstrap.php#L11

Reply:
wtf? O_o

Reply:
https://github.com/symfony/recipes/commit/65dc172421cd7c70e8e46759c02b69faa41a95b4

Reply:
IMHO this should be use in prod... DotEnv I mean